### DISCUSSION TITLE
Hey, this is not an issue directly but does this include the gopro 12 GP-log?

### DISCUSSION THREAD

**Discussion by temtekmedia** [xtremestuff/protune-transforms#15] [General]

**Comment by RikshaDriver** [comment]
GP-Log is basically LogB=400. You will find there are a number of LogB400 transforms using a Rec.709 color space.

However, GoPro recently changed GP-Log to use the WIDE color space as default. For WIDE specific transforms, refer to the [GP-Tune Transform](https://xtremestuff.net/store/gp-tune-transform/)
